test/evergreen/run_format_configs.sh

```diff
@@ -21,20 +21,59 @@ fi
 
 success=0
 failure=0
+running=0
+# Jobs to execute in parallel (hard coded to 8)
```

Comment:
Other parts of our evergreen setup and tooling use the -j <num> notation. I think being consistent there is a good idea when we have different machine sizes (see smp_command in evergreen.yml).

Reply:
Fixed in 913a1ba0e1fca8768e30cd529b12f8703ab90b27